tests/console/x_vt.pm

```diff
@@ -15,10 +15,18 @@ use base "consoletest";
 use strict;
 use warnings;
 use testapi;
+use version_utils 'is_tumbleweed';
 
 sub run {
-    assert_script_run('ps -ef | grep bin/X');
-    assert_screen('xorg-tty');
+    if (script_run("ps -ef | grep bin/X | egrep 'tty7|wayland'") == 1) {
+        if (is_tumbleweed && script_run('ps -ef | grep bin/X | grep tty2') == 0) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. I guess by now also SLE15SP2 behaves like this as there was a recent gnome 3.34 submission

Reply:
I thought the bug is TW live medium specific, I didn't want that some unexpected behaviour will get hidden with generic exception

Reply:
I see. It makes sense that this bug was seen there first but I assume by now it is in Tumbleweed and also in SLE15SP2. I assume your tests can show this anyway.